en/tutorials-and-examples/blog/part-three.rst

```diff
@@ -177,7 +177,7 @@ read if you want re-familiarize yourself with how CakePHP works.
 .. note::
     If you are on Windows remember to use \ instead of /.
 
-You'll need to edit the following in **src/Template/Categories/index.ctp**::
+You'll need to edit the following in **src/Template/Categories/add.ctp**::
```

Comment:
This has already been changed in #3160. Can you cancel the changes for this file ?